src/crispy/StrongHash.h

```diff
@@ -82,7 +82,7 @@ struct StrongHash
 #endif
 };
 
-inline StrongHash::StrongHash(uint32_t a, uint32_t b, uint32_t c, uint32_t d) noexcept:
+inline strong_hash::strong_hash(uint32_t a, uint32_t b, uint32_t c, uint32_t d) noexcept:
 #if defined(STRONGHASH_USE_INTRINSICS)
     StrongHash(Intrinsics::xor128(Intrinsics::load32(a, b, c, d),
```

Comment:
Rename this as well.
also a rename also needs to be done [here](https://github.com/contour-terminal/contour/blob/dad42e223d5cc409de17832fb6ff0b39860a6c3b/src/crispy/StrongHash.h#L96)